src/profile-logic/transforms.js

```diff
@@ -297,7 +291,7 @@ export function getTransformLabels(
   transforms: Transform[]
 ) {
   const { funcTable, libs, stringTable, resourceTable } = thread;
-  const labels = transforms.map(transform => {
+  const labels: string[] = transforms.map(transform => {
```

Comment:
Flow couldn't infer that the inside function is returning strings, even when using `(transform): string =>`. Only typing the return value seemed to help here.